docs/fuse-ufs.8

```diff
@@ -0,0 +1,65 @@
+.\" Copyright (c) 2024 Benjamin Stürz <benni@stuerz.xyz>
+.Dd $Mdocdate$
+.Dt FUSE-UFS 8
+.Os
+.Sh NAME
+.Nm fuse-ufs
+.Nd FUSE3 implementation FreeBSD's UFSv2
+.Sh SYNOPSIS
+.Nm fuse-ufs
+.\" TODO: options
+.Op Fl o Ar options
+.Ar special
+.Ar mountpoint
+.Sh DESCRIPTION
+.Nm
+allows you to mount a FreeBSD UFSv2 filesystem.
+.\" TODO: expand + mention bi-endian support
+
+The following options are available:
+.Bl -tag -width indent
+.It Fl o Ar options
+Pass additional mount options to the kernel.
+.\" TODO: more details
+.It Fl h , -help
+Print a help page and exit.
+.It Fl v , -version
+Print the version of
+.Nm
+and exit.
+.El
+.Sh ENVIRONMENT
+.Ev RUST_LOG
+one of: error, warn, info, debug, trace
+.\" TODO: this looks ugly
+.Sh FILES
+.\" TODO
+.Sh EXIT STATUS
+.Ex -std
+.Sh EXAMPLES
+.\" TODO: how to mount a filesystem
+.Sh SEE ALSO
+.\" TODO
+.Sh HISTORY
+.\" TODO: give credit to Kirk McKusick for UFS
+.Sh AUTHORS
+.An Benjamin Stürz Aq Mt benni@stuerz.xyz
+.An Alan Somers Aq Mt alan@freebsd.org
```

Comment:
Aww, thanks for crediting me.  But you got my email address wrong.
```suggestion
.An Alan Somers Aq Mt asomers@freebsd.org
```

Reply:
Your help was essential for this project, so you'll have to live with the credit :)